tests/wpt/webgl/meta/conformance2/state/gl-object-get-calls.html.ini

```diff
@@ -1,5 +1,5 @@
 [gl-object-get-calls.html]
-  expected: ERROR
+  expected: TIMEOUT
```

Comment:
We should figure out why this is timing out now.

Reply:
Actually there are some (unrelated) buffer drawing calls in the tests that produce GL errors with my integrated GPU, yet work fine for others, including the CI. For the tests in this PR, the crash to timeout changes are due to an error message appearing while running the test, which stops executing the rest of the JS code, while the enum error happens because I've uncommented the failing drawing call during development (and forgot to remove these tests from the PR). I'd be interested if the CI reports any errors other than these, then I do plan to take a look into the buffer implementation and its issues too.